_events/sr2020/competition.md

```diff
@@ -89,5 +91,6 @@ hotels which may be suitable.
 
 [reading-campus-directions]: https://www.rusu.co.uk/contact/find-us/
 [teams-contact]: mailto:teams@studentrobotics.org
+[covid-19]: https://studentrobotics.org/news/2020-03-11-sr2020-covid-19-updates/
```

Comment:
Should we update this to point to the page directly rather than at the news post?